apps/pie-microsite/src/_11ty/shortcodes/tokensTable.js

```diff
@@ -1,4 +1,7 @@
+/* eslint-disable no-trailing-spaces */
 const pieDesignTokens = require('@justeat/pie-design-tokens/dist/tokens.json');
+const pieTokensMetadata = require('../../pieDesignTokensMetadata.json');
+const pieTokenCategories = require('../../pieDesignTokenCategories.json');
```

Comment:
I don't think we necessarily need to prefix all these with `pie`. I think they make as much sense without that. I would probably name these `designTokens`, `tokensMetadata` & `tokensCategories`